package.json

```diff
@@ -52,11 +53,12 @@
     "body-parser": "^1.18.2",
     "cassandra-driver": "^3.5.0",
     "detect-port": "^1.2.2",
+    "drill-client": "^0.1.2",
```

Comment:
It doesn't look like this was used below - could it be removed from the dependencies?